cg/cli/upload/clinical_delivery.py

```diff
@@ -65,12 +65,14 @@ def upload_clinical_delivery(context: click.Context, case_id: str, dry_run: bool
         {"jobs": [str(job_id)]}, config_path=rsync_api.trailblazer_config_path
     )
     analysis_name: str = f"{case_id}_rsync" if is_complete_delivery else f"{case_id}_partial"
+    order_id: int = case.order_id
```

Comment:
Oh wait, this won't work after today's discussion. We are scrapping this column.